mailbox/api/src/test/java/org/apache/james/mailbox/events/ErrorHandlingContract.java

```diff
@@ -297,7 +291,19 @@ default void redeliverShouldNotSendEventsToKeyListeners() {
         eventBus().reDeliver(GROUP_A, EVENT).block();
 
         WAIT_CONDITION
-            .until(() -> assertThat(eventCollector.getEvents()).hasSize(1));
+            .until(() -> eventCollector.getEvents().size() == 1);
```

Comment:
I've just checked, you can replace `.until()` by `untilAsserted()` and all of the assertion lambdas will be intact.

Reply:
Could be... but if you check what we do with `org.awaitility` in the rest of the code, it seems that we do this way. I prefer keeping things consistent in our code if possible.

Do you agree with that?

Reply:
However I agree that it allows me to not use that ugly piece of code I wrote in the `deadLetterContainsOnlyAnEvent` method

Reply:
> Do you agree with that?

yep